vendor/assets/stylesheets/dvl/core/forms.scss

```diff
@@ -327,6 +327,25 @@ input[type="radio"] {
   }
 }
 
+.input_group_date,
+.input_group_time {
+  float: left;
+}
+
+.input_group_date {
+  width: $rhythm * 20;
```

Comment:
Isn't 20 just a random number? What's the point of multiplying by rhythm in this case?